src/main/java/pl/bartek/parking/model/Money.java

```diff
@@ -0,0 +1,63 @@
+package pl.bartek.parking.model;
+
+public class Money {
+
+    private final int value;
+    private final Currency currency;
+
+    private Money(int value, Currency currency) {
+        this.value = value;
+        this.currency = currency;
+    }
+
+    public static Money fromPlnGrosze(int value){
+        return new Money(value, Currency.PLN);
+    }
+
+    public static Money fromPlnZlote(int value){
+        return fromPlnGrosze(value * 100);
+    }
+
+    public Money add(Money other){
+        if(this.currency != other.currency){
+            throw new MoneyOperationException("Can't add money of different currencies!");
+        }
+        return new Money(this.value + other.value, this.currency);
+    }
+
+    public Money multiply(double times){
```

Comment:
Też miałem tutaj z tym ból serca, bo wiem że double z pieniędzmi to zło (dlatego zrobiłem swoją klasę Money z intem). Chętnie posłucham jak dobrze zrobić x 1.5 bez double :)